src/main/java/io/ncbpfluffybear/supserv/items/FarmersHoe.java

```diff
@@ -0,0 +1,108 @@
+package io.ncbpfluffybear.supserv.items;
+
+import io.github.thebusybiscuit.slimefun4.core.attributes.DamageableItem;
+import io.github.thebusybiscuit.slimefun4.core.attributes.NotPlaceable;
+import io.github.thebusybiscuit.slimefun4.core.handlers.ItemUseHandler;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import io.github.thebusybiscuit.slimefun4.implementation.items.SimpleSlimefunItem;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import me.mrCookieSlime.Slimefun.cscorelib2.protection.ProtectableAction;
+import org.bukkit.CropState;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.block.data.Ageable;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.Nonnull;
+import java.util.Arrays;
+import java.util.List;
+
+public class FarmersHoe extends SimpleSlimefunItem<ItemUseHandler> implements NotPlaceable, DamageableItem {
+
+  static final List<Material> BREAKONLY = Arrays.asList(Material.MELON, Material.PUMPKIN);
+  static final List<Material> FARMABLES = Arrays.asList(Material.DIRT, Material.GRASS_BLOCK, Material.COARSE_DIRT);
+  static final List<Material> TALLPLANTS = Arrays.asList(Material.SUGAR_CANE, Material.CACTUS, Material.BAMBOO,
+          Material.CHORUS_PLANT);
+
+  public FarmersHoe(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe) {
+    super(category, item, recipeType, recipe);
+
+  }
+
+  @Nonnull
+  @Override
+  public ItemUseHandler getItemHandler() {
+    return e -> {
+
+      Player p = e.getPlayer();
+      ItemStack item = e.getItem();
+
+
+      if (!e.getClickedBlock().isPresent()) {
+        return;    // Check if clicked block is null > return
+      }
+      Block b = e.getClickedBlock().get();
+      if (FARMABLES.contains(b.getType())) {
+        return;
+      }
+      e.cancel();
+      if (!SlimefunPlugin.getProtectionManager().hasPermission(e.getPlayer(), b,
```

Comment:
According to Google's style guide, you can have lines up to 100 characters. I'm pretty sure this doesn't exceed that limit, so you can put it back in one line.